packages/salesforcedx-vscode-soql/test/vscode-integration/commands/soqlFileCreate.test.ts

```diff
@@ -21,8 +21,6 @@ describe('soqlOpenNew should', () => {
   beforeEach(() => {
     sb = createSandbox();
     telemetryStub = sb.stub(telemetryService, 'sendCommandEvent') as SinonStub;
-    editorOpened = sb.stub();
```

Comment:
this is no longer called, since we do not interact with the openTextDocument() api.  Therefore, removed from test and associated expectations.